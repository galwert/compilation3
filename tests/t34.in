int mainFunc(int main, bool killSwitch)
{
    if (killSwitch)
    {
        return main;
    }
    else
    {
        return 0;
    }

    main = main if (killSwitch) else 0;
    return main;
}

void main()
{
    return;
}