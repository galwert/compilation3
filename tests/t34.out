---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
main INT -1
killSwitch BOOL -2
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
mainFunc (INT,BOOL)->INT 0
main ()->VOID 0
